Source/Dafny/Compilers/SinglePassCompiler.cs

```diff
@@ -1027,7 +1027,9 @@ public class TypeArgumentInstantiation {
       ConcreteSyntaxTree wStmts);
     protected abstract void EmitApplyExpr(Type functionType, Bpl.IToken tok, Expression function, List<Expression> arguments, bool inLetExprBody, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts);
     protected virtual bool TargetLambdaCanUseEnclosingLocals => true;
-    protected abstract ConcreteSyntaxTree EmitBetaRedex(List<string> boundVars, List<Expression> arguments, List<Type> boundTypes, Type resultType, Bpl.IToken resultTok, bool inLetExprBody, ConcreteSyntaxTree wr, ConcreteSyntaxTree wStmts);
+    protected abstract ConcreteSyntaxTree EmitBetaRedex(List<string> boundVars, List<Expression> arguments,
+      List<Type> boundTypes, Type resultType, Bpl.IToken resultTok, bool inLetExprBody, ConcreteSyntaxTree wr,
+      ConcreteSyntaxTree wStmts, out ConcreteSyntaxTree wrStmts);
```

Comment:
These parameter names definitely need to be longer, or at least to have a comment on the method signature. I have no idea what the intended difference between `wStmts` and `wrStmts` is :)